package-name-map/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "package-name-map",
```

Comment:
"private": true?

Reply:
I was hoping to publish this to use in other prospective libraries/tools.

Reply:
Maybe publish it before someone takes the name? 😜